kong-ngx-build

```diff
@@ -332,32 +344,44 @@ main() {
   echo ""
   echo "Usage: $0 [options...] -p <prefix> --openresty <openresty_ver> --openssl <openssl_ver>"
   echo ""
   echo "Required arguments:"
-  echo "  -p, --prefix                  location where components should be installed to"
-  echo "      --openresty               version of OpenResty to build, such as 1.13.6.2"
-  echo "      --openssl                 version of OpenSSL to build, such as 1.1.1b"
+  echo "  -p, --prefix <prefix>            Location where components should be installed."
+  echo "      --openresty <openresty_ver>  Version of OpenResty to build, such as 1.13.6.2."
+  echo "      --openssl <openssl_ver>      Version of OpenSSL to build, such as 1.1.1b."
+  echo ""
   echo "Optional arguments:"
-  echo "      --no-build-cache          disable compilation caching and re-download source code to"
-  echo "                                build from scratch"
-  echo -e "                                \033[1;31mWARNING:\033[0m this removes everything inside the work directory"
-  echo "      --no-artifact-cache       disable artifact caching and re-install all the softwares"
-  echo -e "                                \033[1;31mWARNING:\033[0m this removes everything inside the prefix directory"
-  echo "      --no-openresty-patches    do not apply openresty-patches while compiling OpenResty, patching is"
-  echo "                                enabled by default"
-  echo "      --luarocks                version of LuaRocks to build, such as 3.0.4. if absent LuaRocks"
-  echo "                                will not be built"
-  echo "      --add-module              path to additional NGINX module to be built, this option can be repeated"
-  echo "                                and will be passed to NGINX's configure in the order they were specified"
-  echo "      --debug                   disable compile time optimizations and memory pooling for NGINX,"
-  echo "                                LuaJIT and OpenSSL to help debugging"
-  echo "      -j, --jobs                concurrency level to use when building, defaults to number of CPU"
-  echo "                                cores available ($NPROC)"
-  echo "      --work                    the working directory to use while compiling, defaults to \"work\""
-  echo "  -h, --help                    show this usage"
+  echo "      --no-build-cache             Build from scratch."
+  echo -e "                                   (\033[1;31mWARNING:\033[0m this removes everything inside the work directory)"
+  echo ""
+  echo "      --no-artifact-cache          Disable artifact caching and re-install all the softwares."
+  echo -e "                                   (\033[1;31mWARNING:\033[0m this removes everything inside the prefix directory)"
+  echo ""
+  echo "      --no-openresty-patches       Do not apply openresty-patches while compiling OpenResty."
+  echo "                                   (Enabled by default)"
```

Comment:
This seems slightly ambigious. Maybe the original "patching is enabled by default" is better. Otherwise it may sounds like `--no-openresty-patches` is enabled by default.

Reply:
Oops, definitely a copy-paste gone wrong here, I don't think I would have worded this like so intentionally, since it is indeed very confusing. Good catch.